backend/src/main/java/de/neuefische/booklibrary/backend/BookService.java

```diff
@@ -15,4 +15,8 @@ public class BookService {
     public List<Book> getAllBooks() {
         return bookRepository.findAll();
     }
+
+    public Book saveBook(Book book) {
+        return bookRepository.save(book);
```

Comment:
Maybe use .insert(book). If the ID already exists, insert throws an error instead of replacing the existing object.

Reply:
I see that you want it to work that way, then it's fine